### ISSUE TITLE
startCase

### ISSUE THREAD

**Issue by steveoh** [dgilland/pydash#60]
I'm looking for an python implementation of [startCase.](https://lodash.com/docs#startCase)

Thoughts on how this might be done and should it end up in here?

PS: it was [added to lodash Feb 1, 2015](https://github.com/lodash/lodash/pull/908/files)

**Comment by dgilland** [comment]
Already have you covered:

- http://pydash.readthedocs.org/en/latest/api.html#pydash.strings.start_case
- https://github.com/dgilland/pydash/blob/develop/pydash/strings.py#L1327

**Comment by steveoh** [comment]
ha, i must have search for startCase... You rock!

**Comment by steveoh** [comment]
So, I don't think you're start_case [is working properly](https://github.com/lodash/lodash/issues/1176).

```python
>>> from pydash.strings import start_case
>>> start_case('XMLHttp')
'Xml Http'
```

**Comment by dgilland** [comment]
You're right. It shouldn't modify the case of anything other than the first letter of each word. Will push out a fix soon.

**Comment by dgilland** [comment]
Bug fix as been released as `v3.4.2`: https://pypi.python.org/pypi/pydash/3.4.2

Thanks again for finding the bug! :+1:
